src/status_im/contexts/communities/actions/community_rules/view.cljs

```diff
@@ -1,14 +1,19 @@
 (ns status-im.contexts.communities.actions.community-rules.view
   (:require
+    [quo.components.buttons.slide-button.utils :as slide-button-utils]
     [quo.core :as quo]
     [react-native.core :as rn]
+    [react-native.platform :as platform]
     [status-im.contexts.communities.actions.community-rules.style :as style]
     [utils.re-frame :as rf]))
 
 (defn view
   [id]
-  (let [rules (rf/sub [:communities/rules id])]
-    [rn/view {:style style/community-rule}
+  (let [rules                 (rf/sub [:communities/rules id])
+        android-bottom-margin (if platform/android? 44 0)
+        rules-bottom-space    (+ (slide-button-utils/get-dimensions nil :size-48 :track-height)
+                                 android-bottom-margin)]
```

Comment:
I would recommend storing the result of the `slide-button-utils/get-dimensions` call in a variable for better readability. Additionally, renaming `rules-bottom-space` to `rules-bottom-margin` would make the code clearer, as it accurately represents its purpose as the bottom margin.

```suggestion
slide-button-height   (slide-button-utils/get-dimensions nil :size-48 :track-height)
rules-bottom-margin   (+ slide-button-height android-bottom-margin)]
```

Reply:
oh, thats cool. I will do it. Tks!